tests/e2e/greenpeace-media.spec.js

```diff
@@ -0,0 +1,75 @@
+import {test, expect} from './tools/lib/test-utils.js';
+import {
+  searchImages,
+  deleteImageFromLibrary,
+  waitForLibraryLoad,
+} from './tools/lib/media-library.js';
+
+const IMAGE_1 = 'GP0STXWME';
+const IMAGE_2 = 'GP0STXWZH';
+
+test.useAdminLoggedIn();
+
+test.describe('Greenpeace Media tests', () => {
+  test('import an image to the Library using a search term', async ({page}) => {
+    // Make sure to close any alerts.
+    page.on('dialog', dialog => dialog.accept());
+
+    // Go to the Media Library and wait for the page to load.
+    await page.goto('./wp-admin/admin.php?page=media-picker');
+    await waitForLibraryLoad(page);
+
+    // Search for an image to import.
+    await searchImages(page, 'Rainbow Warrior');
+    const image = page.locator('.picker-list > li')
+      .filter({hasNotText: 'Added to Media Library'})
+      .first();
+    await image.click();
+    await expect(image).toHaveClass('is-selected');
+    const sidebar = page.locator('.archive-picker-sidebar');
+    await expect(sidebar).toBeVisible();
+
+    // Import the image.
+    await sidebar.getByRole('button', {name: 'Import to Library'}).click();
+    await expect(sidebar.getByText('Processing...')).toBeVisible();
+    await expect(sidebar.getByText('Added to Library')).toBeVisible();
+
+    // Delete the image from the library.
```

Comment:
Shouldn't we check that the image also exists in `/wp-admin/upload.php` before deleting it?

Reply:
Actually for deleting an image, I had another possible implementation in mind, in which we could get the WordPress image id (from the data attributes) and go directly to the associated media page to delete it. I didn't go for it because it felt less "end to end" but what do you think?

Reply:
I like the idea :)